sdktests/server_side_persistence_dynamodb.go

```diff
@@ -0,0 +1,246 @@
+package sdktests
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"math"
+	"strconv"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/launchdarkly/sdk-test-harness/v2/servicedef"
+)
+
+const (
+	// Schema of the DynamoDB table
+	tablePartitionKey = "namespace"
```

Comment:
Since these are all in the same package, you could access, say, `tableName` from the Redis file or whatever. It could be worth prefixing these to make them unique. 

Better yet would be refactoring these into dedicated packages, that way they can't start accidentally sharing constants. But that would be a fair amount of work. 

I'm surprised `initedKey` isn't being used in Redis impl (and is being used here)

Reply:
Prefixed the dynamodb ones, and then moved the init one to the base file (I know that doesn't change anything, it's just an implication). Also changed its name and updated the usage in other places.